bot/exts/filters/filtering.py

```diff
@@ -387,16 +387,19 @@ async def _filter_message(self, msg: Message, delta: Optional[int] = None) -> No
                                 log.trace(f"Offensive message {msg.id} will be deleted on {delete_date}")
 
                         stats = self._add_stats(filter_name, match, msg.content)
-                        await self._send_log(filter_name, _filter, msg, stats, reason)
 
-                        # If the filter reason contains `[autoban]`, we want to auto-ban the user
+                        # If the filter reason contains `[autoban]`, we want to auto-ban the user.
+                        # Also pass this to _send_log so mods are not pinged filter matches that are auto-actioned
                         autoban = reason and "[autoban]" in reason.lower()
                         if not autoban and filter_name == "filter_invites" and isinstance(result, dict):
                             autoban = any(
                                 "[autoban]" in invite_info["reason"].lower()
                                 for invite_info in result.values()
                                 if invite_info.get("reason")
                             )
+
+                        await self._send_log(filter_name, _filter, msg, stats, reason, autoban=autoban)
```

Comment:
Might be better to instead rename the keyword to `autoban` for consistency.
```suggestion
                        await self._send_log(filter_name, _filter, msg, stats, reason, auto_ban=autoban)
```

Reply:
https://github.com/python-discord/bot/pull/2196/commits/11f329f6847c0e5f97dac248507bbd2970804f43